google-cloudevent-types/src/main/java/com/google/events/cloud/audit/v1/AuditLog.java

```diff
@@ -0,0 +1,122 @@
+package com.google.events.cloud.audit.v1;
+
+import java.util.Map;
+
+/**
+ * The log entry payload, which is always an AuditLog for Cloud Audit Log events.
+ */
+public class AuditLog {
+    private AuthenticationInfo authenticationInfo;
+    private AuthorizationInfo[] authorizationInfo;
+    private Map<String, Object> metadata;
+    private String methodName;
+    private Long numResponseItems;
+    private Map<String, Object> request;
+    private RequestMetadata requestMetadata;
+    private ResourceLocation resourceLocation;
+    private String resourceName;
+    private Map<String, Object> resourceOriginalState;
+    private Map<String, Object> response;
+    private Map<String, Object> serviceData;
+    private String serviceName;
+    private Status status;
+
+    /**
+     * Authentication information.
+     */
+    public AuthenticationInfo getAuthenticationInfo() { return authenticationInfo; }
+    public void setAuthenticationInfo(AuthenticationInfo value) { this.authenticationInfo = value; }
```

Comment:
Can we get qt to generate setter documentation as well as getter docs?

Reply:
It looks like the generator doesn't provide comments to setters :\ I'll file an issue and see if we can add it.

https://github.com/googleapis/google-cloudevents-java/issues/9